jobs/keepalived/templates/bin/utils/openstack/generate_token

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#sourcing job properties to bring openstack credential in env variables
+# shellcheck disable=SC1091
+source /var/vcap/jobs/keepalived/bin/job_properties.sh
+jq=/var/vcap/packages/jq/bin/jq
```

Comment:
since we are using this in multiple places, will it help to place in the job_properties file itself?

Reply:
done